src/libraries/illink-sharedframework.targets

```diff
@@ -30,8 +30,20 @@
       <ILLinkArgs>$(ILLinkArgs) --nowarn $(LinkerNoWarn)</ILLinkArgs>
     </PropertyGroup>
 
-     <!-- Retrieve CoreLib's targetpath via GetTargetPath as it isn't binplaced yet. -->
-    <MSBuild Projects="$(CoreLibProject)"
+    <!-- Retrieve CoreLib's path as it isn't binplaced alongside the libraries. -->
```

Comment:
@ViktorHofer - is there a tracking issue for this work?

Reply:
https://github.com/dotnet/runtime/issues/43095